### DISCUSSION TITLE
We should continue writing the book

### DISCUSSION THREAD

**Discussion by WeetHet** [hecrj/iced#1705] [Ideas]
Iced book is incomplete for a long time now. Maybe it's time we get back to it?

**Comment by DaraJKong** [comment]
I totally agree! Since Iced is starting to be used by Systems76 for their Cosmic DE, more and more people will be interested in using Iced to make applications for Cosmic. When the project gets a lot of attention, documentation becomes increasingly important for the newcomers to get started quickly.

The way I see it (just my opinion) is contributors who like me, do not have the knowledge to contribute to features/bug fixes, could at least work on the book. This way, the core team doesn't need to slow down the development of Iced, and the book can at least get to a decent starting point for anyone. When the API actually gets more stable, then the book could be refined and made official.

**Comment by 13r0ck** [comment]
Hecrj has commented on the state or iced documentation [on the iced discord](https://discord.com/channels/628993209984614400/1021828532189528094/1070054543963979786).
The focus seems to be on community docs at the moment. The [system76 docs](https://github.com/system76/docs) could be a good place to host community documentation.

**Comment by elaforma** [comment]
Speaking as a newcomer: Given the current state of the book, I think it shouldn't be linked to so prominently on the website. No offence, but it is rather useless right now. It links to some other (good!) resources, but those should be recommended directly by the website.
